CandidateExercise/ConfigurationExcerciseTests/ConfigTests.cs

```diff
@@ -22,12 +25,38 @@ public ConfigTests()
             XmlConfigurator.Configure();
         }
 
-        [TestInitialize]
-        private void TestFrameworkInitialization()
+
+        List<String> keyList;
```

Comment:
Can you explicitly declare your variable scope, please?

I imagine you could also further limit the potential misuse of these variables with Readonly/Const.